datasrc/network.py

```diff
@@ -56,7 +57,8 @@
 Flags = [
 	Flags("PLAYERFLAG", PlayerFlags),
 	Flags("GAMEFLAG", GameFlags),
-	Flags("GAMESTATEFLAG", GameStateFlags)
+	Flags("GAMESTATEFLAG", GameStateFlags),
+	Flags("CHARACTERFLAG", CharacterFlags)
```

Comment:
```suggestion
	Flags("CHARACTERFLAG", CharacterFlags),
```
Next diff doesn't need to touch two lines.

Reply:
why is that important? I think its more clear without that comma

Reply:
easier to append a new flag, easier to resolve conflicts.